### DISCUSSION TITLE
issues forms missing preview feature for entire post

### DISCUSSION THREAD

**Discussion by lukastribus** [community/community#4275] [Projects and Issues]
The new issue forms do not have a preview feature for the entire post, like normal issues. The preview feature is limited to single input boxes, but that is not very useful.

Please consider adding a full-port preview feature.

Thanks,
Lukas

**Comment by lukehefson** [comment]
> but that is not very useful.

Hi @lukastribus 👋. Could you give us a little more info on why you don't think it's useful? What is it about a full-port preview feature that would make issue forms better for you than preview-per-input?

**Reply by TimWolla** [reply]
> about a full-port preview

I believe that should've read 'full-post preview'. Could it be that this caused confusion? Because I believe that a full-post preview is obviously superior:

1. The user can easily see the to-be-created issue in its full glory, including the parts that are formatted as code and come without the markdown editor and thus without a preview.
2. It only requires the user to click a single button after filling in the fields vs. needing to click preview for each field, ultimately saving clicks and time.

I can't think of a reason where looking at a separate preview of each field would be preferable to simply seeing the issue as it will look after pressing submit.

**Reply by lukastribus** [reply]
Yes, this should have been "full post preview".

As a user filling out the template, I wanna check if the entire post/report looks OK and doesn't have any obvious stylistic issues.

The last thing I want to do is to check 5 or 6 different input boxes individually, because a) its a lot of unnecessary back and forth clicking on preview buttons and going back and b) it doesn't show me the big picture (of the entire post) - which really defeats the purpose.

thanks,
lukas

**Reply by lukehefson** [reply]
We took the initial decision to not show a full preview because the current, familiar design paradigm of <kbd>Write | Preview</kbd> boxes include a markdown toolbar and attachment section that we knew we'd _only want to apply to particular types of input_ (e.g. no attachments for `input` or markdown toolbar for `dropdown`).

If the preview was 'global' we would need to rebuild an entirely new user experience for showing which input types need a markdown toolbar and which need an attachments section.

However, it's certainly something we'd like to investigate in the future as I do agree that it would be preferable to get a single preview of what a submitted issue will look like!

**Reply by lukastribus** [reply]
Yes, this was enough because the entire post was markdown encoded in a single, big input field. Now with issue forms this is no longer the case.
